src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -139,7 +120,7 @@ class WorkflowView extends Component {
       const inputsOutputs = this.createIOLists(validationResponse, ioDefinitions)
 
       this.setState({
-        isFreshData: true, config, entityMetadata, inputsOutputs, ioDefinitions, firecloudRoot,
```

Comment:
Does this mean we can remove firecloudRoot from config.js and the json files?

Reply:
I think so.